### DISCUSSION TITLE
Clarification on cache support

### DISCUSSION THREAD

**Discussion by donferi** [cloudflare/next-on-pages#457] [Q&A]
Hello, I've seen that a recent PR got merged to add support for next cache. I can't find on the docs exactly what is supported. From my understanding using `revalidateTag`, and `revalidate` on fetch are working as expected.

I'm still getting a `The 'cache' field on 'RequestInitializerDict' is not implemented.` error when trying to send a fetch request with `cache: 'no-store'`. I understand workerd doesn't support this but I thought next-on-pages would patch fetch. Am I missing something? Are there any docs / examples around this? Thank you so much!

**Comment by james-elicx** [comment]
Hello, please could you share which version of `@cloudflare/next-on-pages`, Next.js, and Vercel CLI you are using?

You can run `npx @cloudflare/next-on-pages --info` and it should provide a nice list of information about your project.

**Reply by donferi** [reply]
Sure thing!

```
System:
	Platform: darwin
	Arch: arm64
	Version: Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:20 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6000
	CPU: (10) arm64 Apple M1 Pro
	Memory: 16 GB
	Shell: /nix/store/rxj0jxd6j6n7wwpqj8k843jha6xs9vls-zsh-5.9/bin/zsh
Binaries:
	Node: 18.13.0
	Bun: 0.1.4
	pnpm: 8.7.5
	Yarn: N/A
	npm: 8.19.3
Package Manager Used: pnpm
Relevant Packages:
	@cloudflare/next-on-pages: 1.6.0
	vercel: N/A
	next: 13.4.19
```

**Reply by donferi** [reply]
Hey @james-elicx sorry for the bump, I just tried again with a new project and I'm hitting the same error. I must be missing something I just don't know what it can be. Im not doing anything interesting on my test, just a regular RSC with `fetch` passing a `cache: force-cache` option. In a different page I'm also trying a fetch with `next { revlidate: 10 }`
